javalanglib/src/main/scala/java/lang/Character.scala

```diff
@@ -519,6 +520,362 @@ object Character {
+    val MAHJONG_TILES = addUnicodeBlock("MAHJONG_TILES", "Mahjong Tiles", 0x1F000, 0x1F02F)
+    val DOMINO_TILES = addUnicodeBlock("DOMINO_TILES", "Domino Tiles", 0x1F030, 0x1F09F)
+    val PLAYING_CARDS = addUnicodeBlock("PLAYING_CARDS", "Playing Cards", 0x1F0A0, 0x1F0FF)
+    val ENCLOSED_ALPHANUMERIC_SUPPLEMENT = addUnicodeBlock("ENCLOSED_ALPHANUMERIC_SUPPLEMENT", "Enclosed Alphanumeric Supplement", 0x1F100, 0x1F1FF)
+    val ENCLOSED_IDEOGRAPHIC_SUPPLEMENT = addUnicodeBlock("ENCLOSED_IDEOGRAPHIC_SUPPLEMENT", "Enclosed Ideographic Supplement", 0x1F200, 0x1F2FF)
+    val MISCELLANEOUS_SYMBOLS_AND_PICTOGRAPHS = addUnicodeBlock("MISCELLANEOUS_SYMBOLS_AND_PICTOGRAPHS", "Miscellaneous Symbols and Pictographs", 0x1F300, 0x1F5FF)
+    val EMOTICONS = addUnicodeBlock("EMOTICONS", "Emoticons", 0x1F600, 0x1F64F)
+    val TRANSPORT_AND_MAP_SYMBOLS = addUnicodeBlock("TRANSPORT_AND_MAP_SYMBOLS", "Transport and Map Symbols", 0x1F680, 0x1F6FF)
+    val ALCHEMICAL_SYMBOLS = addUnicodeBlock("ALCHEMICAL_SYMBOLS", "Alchemical Symbols", 0x1F700, 0x1F77F)
+    val CJK_UNIFIED_IDEOGRAPHS_EXTENSION_B = addUnicodeBlock("CJK_UNIFIED_IDEOGRAPHS_EXTENSION_B", "CJK Unified Ideographs Extension B", 0x20000, 0x2A6DF)
+    val CJK_UNIFIED_IDEOGRAPHS_EXTENSION_C = addUnicodeBlock("CJK_UNIFIED_IDEOGRAPHS_EXTENSION_C", "CJK Unified Ideographs Extension C", 0x2A700, 0x2B73F)
+    val CJK_UNIFIED_IDEOGRAPHS_EXTENSION_D = addUnicodeBlock("CJK_UNIFIED_IDEOGRAPHS_EXTENSION_D", "CJK Unified Ideographs Extension D", 0x2B740, 0x2B81F)
+    val CJK_COMPATIBILITY_IDEOGRAPHS_SUPPLEMENT = addUnicodeBlock("CJK_COMPATIBILITY_IDEOGRAPHS_SUPPLEMENT", "CJK Compatibility Ideographs Supplement", 0x2F800, 0x2FA1F)
+    val TAGS = addUnicodeBlock("TAGS", "Tags", 0xE0000, 0xE007F)
+    val VARIATION_SELECTORS_SUPPLEMENT = addUnicodeBlock("VARIATION_SELECTORS_SUPPLEMENT", "Variation Selectors Supplement", 0xE0100, 0xE01EF)
+    val SUPPLEMENTARY_PRIVATE_USE_AREA_A = addUnicodeBlock("SUPPLEMENTARY_PRIVATE_USE_AREA_A", "Supplementary Private Use Area-A", 0xF0000, 0xFFFFF)
+    val SUPPLEMENTARY_PRIVATE_USE_AREA_B = addUnicodeBlock("SUPPLEMENTARY_PRIVATE_USE_AREA_B", "Supplementary Private Use Area-B", 0x100000, 0x10FFFF)
+
+    ////////////////
+    // End Generated
+    ////////////////
+    // scalastyle:on line.size.limit
+
+    // Additional JVM Aliases
+    blocksByNormalizedName.put("Greek and Coptic".toLowerCase, GREEK)
+    blocksByNormalizedName.put("GreekandCoptic".toLowerCase, GREEK)
+    blocksByNormalizedName.put("GREEK_AND_COPTIC".toLowerCase, GREEK)
+    blocksByNormalizedName.put("TaiLe".toLowerCase, TAI_LE)
+    blocksByNormalizedName.put("Combining Diacritical Marks for Symbols".toLowerCase, COMBINING_MARKS_FOR_SYMBOLS)
+    blocksByNormalizedName.put("CombiningDiacriticalMarksforSymbols".toLowerCase, COMBINING_MARKS_FOR_SYMBOLS)
```

Comment:
This seems to hard code the logic for alias building. Consider factoring this into a helper.

Reply:
👍